### ISSUE TITLE
Notifications for releases are not listed

### ISSUE THREAD

**Issue by rEnr3n** [meiji163/gh-notify#83]
I'm getting these lines every second until the UI appears.
```
$ gh notify
.unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
unexpected end of JSON input
All caught up!
```

I could see notifications from issues I'm subscribed at but not releases.

**Comment by LangLangBart** [comment]
> I'm getting these lines every second until the UI appears.


Could you add `-o xtrace` and `PS4 …` to the script and share the result so I can narrow down in which function the issue appears and try to find the cause?




```diff
diff --git a/gh-notify b/gh-notify
index 5c58399..fbae009 100755
--- a/gh-notify
+++ b/gh-notify
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
-set -o errexit -o nounset -o pipefail
+set -o errexit -o nounset -o pipefail -o xtrace
 # https://www.gnu.org/software/bash/manual/bash.html#The-Set-Builtin

+PS4='+$(date +%Y-%m-%d:%H:%M:%S) ${BASH_SOURCE:-}:${FUNCNAME[0]:-}:L${LINENO:-}:  '
 # ====================== Infos =======================

 # https://docs.github.com/en/rest/activity/notifications

```

**Comment by rEnr3n** [comment]
I don't see the issue anymore. It might have been on github's end for a short while or because I marked all the notifications as done (through the browser).

Can your suggestion be added as a new flag? Perhaps -v for verbose.

**Comment by LangLangBart** [comment]
Okay, I made a PR with a new environment variable which will capture everything into a debug file, but
still allow you to use this tool normally. The only difference is that at the end, it prints the
path to the file which contains all debug information from `xtrace` as well as `stdout` and `stderr`.

If you want to **try it out**, that would be great; otherwise, it will remain as a PR till I have tested it sufficiently.
